heartbeat/monitors/active/http/http.go

```diff
@@ -138,6 +138,7 @@ func newRoundTripper(config *Config, tls *transport.TLSConfig) (*http.Transport,
 		Proxy:             proxy,
 		Dial:              dialer.Dial,
 		DialTLS:           tlsDialer.Dial,
+		TLSClientConfig:   tls.ToConfig(),
```

Comment:
Can you add this change to some other packages, please?

- `libbeat/output/elasticsearch`
- `metricbeat/helper`

Reply:
@urso I've done so 4 additional spots in 5a4e09c

Reply:
Thank you.